README.md

```diff
@@ -3,35 +3,82 @@
- - Restart any services such as `make lms-restart` for applying changes
- - Pick a user such as enterprise_learner_1 to use as learner account (password is the same as the test edx account documented on devstack)
- - Go to http://localhost:8734/test-enterprise/ to browse the application for the `test-enterprise` customer
+
+## Setup enterprise stack
+
+An enterprise portal will need a couple of roles: The Enterprise customer, and at least one learner account.
+We also need the edx-enterprise Django module to be running inside of the LMS (which is started by devstack). Refer to [Devstack Doc](https://github.com/edx/devstack) for details. This is what serves endpoints used by the application.
+
+* Setup devstack if not already and at least have the `lms` service running. Typically `make dev.up` or `make dev.nfs.up` will do it
+* Ensure you can browser to LMS at least, http://localhost:18000
+
+* Sync edx-enterprise into the src/ folder of your work folder (e.g. ~/work/src/edx-enterprise).
+* Ensure this env var is set in your env for the docker mount to be located correctly, e.g.,: `DEVSTACK_WORKSPACE=/Users/$USER/work` , in your shell config file
+* Either start new shell or run `exec "$SHELL"` for changes to take effect in the shell
+* Start backend services needed (refer to .env.development file). Refer to [Devstack Doc](https://github.com/edx/devstack) for details
+* Now in your devstack folder run
+  ```
+  $ make lms-shell
+  $ pip install -e /edx/src/edx-enterprise to uninstall the published edx-enterprise package and instead install your local copy from src.
+  ```
+If all worked you should be able to navigate to http://localhost:18000/admin/enterprise/
+
+But wait, it wont' work yet. As long as you get some reasonable response here, you are good (probably a login page)
+
+
+## Setup test users and data
+
+Next setup a `test-enterprise` customer who will have learners associated with it (Details at [The Enterprise management commands](https://github.com/edx/edx-enterprise/blob/master/enterprise/management/commands/seed_enterprise_devstack_data.py#L47)):
+
+ - Ensure the Enterprise Integration flag is enabled in devstack [see this link](https://github.com/edx/edx-platform/blob/master/lms/envs/devstack.py#L326). Set the flag `ENABLE_ENTERPRISE_INTEGRATION` to True
+ - From the devstack directory, restart lms using `make lms-restart` for applying changes
+ - Seed test data:
+   - Go into the lms shell:
+   ```$ cd <devstack_dir>
+      $ make lms-shell
+      $ ./manage.py lms seed_enterprise_devstack_data
+   ```
+ - Go to http://localhost:18000/admin/enterprise/ and login as edx/edx
+ - If you can see an `Enterprise` section in the admin page you are all set so far, otherwise stop and ask someone!
+ - Next, ensure the Learner portal is enable for this Enterprise user by checking the box `Enable Learner Portal` in this page http://localhost:18000/admin/enterprise/enterprisecustomer/ and clicking on 'Test-enterprise' customer
```

Comment:
I think this could be worded a bit more clearly. 
The way this reads is:
- there is a checkbox on the enterprisecustomer endpoint screen, but there isn't and I was looking around for a surprising amount of time.

What it should probably say is: "navigate to http://localhost:18000/admin/enterprise/enterprisecustomer/, click on 'Test-enterprise' and then check the box next to `Enable Learner Portal`, scroll down and click save at the bottom"